bin/compile

```diff
@@ -4,21 +4,24 @@
 BUILD_DIR=$1
 CACHE_DIR=$2
 
-WKHTMLTOPDF_URL="https://downloads.wkhtmltopdf.org/0.12/0.12.4/wkhtmltox-0.12.4_linux-generic-amd64.tar.xz"
-XZ_FILENAME=$CACHE_DIR/$(basename $WKHTMLTOPDF_URL)
-WKHTMLTOPDF_PATH=$CACHE_DIR/wkhtmltox
+WKHTMLTOPDF_URL="https://github.com/wkhtmltopdf/wkhtmltopdf/releases/download/0.12.5/wkhtmltox_0.12.5-1.xenial_amd64.deb"
+DEB_FILENAME=$CACHE_DIR/$(basename $WKHTMLTOPDF_URL)
+EXTRACTED_DATA_TAR="$(pwd)/data.tar.xz"
+WKHTMLTOPDF_PATH=$CACHE_DIR/usr/local
 DESTINATION=$BUILD_DIR/bin  # will be `/app/bin` inside dyno
 
 
 # Download and extract wkhtmltopdf
 if [ ! -d "$WKHTMLTOPDF_PATH" ]; then
 	echo '-----> Downloading wkhtmltopdf'
-	mkdir -p $(dirname $XZ_FILENAME)
-	wget --quiet --continue --output-document="$XZ_FILENAME" "$WKHTMLTOPDF_URL"
+	mkdir -p $(dirname $DEB_FILENAME)
+	wget --quiet --continue --output-document="$DEB_FILENAME" "$WKHTMLTOPDF_URL"
 
 	echo '-----> Extracting wkhtmltopdf'
-	tar --xz --extract --file "$XZ_FILENAME" --directory "$CACHE_DIR"
-	rm "$XZ_FILENAME"
+	ar -xv $DEB_FILENAME
+
+	tar -xvf "$EXTRACTED_DATA_TAR" -C "$CACHE_DIR"
```

Comment:
I did not understand why you create this file.

Reply:
Because of how wkhtmltopdf distributes their packages now, I had to first use the `ar` utility to extract the `.deb` archive and then extract the resulting `data` tape archive to obtain the wkhtmltopdf binaries. Does this answer your question? I'll be happy to elaborate further.